ElasticUI Yarn2 support?

So I've converted a basic react app (with typescript CRA template) over to be a yarn 2 app using `yarn set berry`

I then installed elastic ui using following commands from github page:
`yarn add @elastic/eui @elastic/datemath moment prop-types`

When I add `EuiPageTemplate` and necessary child controls to the page, I get compilation errors that look like this:
```
./.yarn/unplugged/@elastic-eui-virtual-1b2cf50b71/node_modules/@elastic/eui/es/components/markdown_editor/plugins/markdown_link_validator.js
Module not found: @elastic/eui tried to access unist-util-visit, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.
```

I'm not sure EUI claims to support being used from within a yarn2 project, but if it should work.... does anyone have any workarounds for this problem? I attempted to install that, but it seemed like it just changed the error slightly.

--- a/package.json
+++ b/package.json
@@ -83,6 +83,7 @@
     "tabbable": "^3.0.0",
     "text-diff": "^1.0.1",
     "unified": "^9.2.0",
+    "unist-util-visit": "^2.0.3",
     "url-parse": "^1.5.0",
     "uuid": "^8.3.0",
     "vfile": "^4.2.0"

